.github/workflows/build.yml

```diff
@@ -8,38 +8,38 @@ jobs:
     strategy:
       matrix:
         platform:
-        - [linux, ubuntu-latest]
-        - [darwin, macos-latest]
+          - [linux, ubuntu-latest]
+          - [darwin, macos-latest]
 
     runs-on: ${{matrix.platform[1]}}
 
     steps:
-    - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-    - name: Install Node
-      uses: actions/setup-node@v1
-      with:
-        node-version: 14.x
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
 
-    - name: 'Build Node with corepack master'
-      run: |
-        yarn install --immutable
-        yarn pack
-        git clone -b mael/pmm --depth=1 https://github.com/arcanis/node.git node && cd node
-        git config user.name 'John Doe'
-        git config user.email 'john@example.org'
-        rm -rf deps/corepack && tar xvf ../package.tgz && mv package deps/corepack
-        git add .
-        if ! git diff-index --quiet HEAD; then
-          git commit -m 'Updates corepack'
-          git show --name-status
-        fi
-        ./configure
-        DISTTYPE=nightly DATESTRING=YYYY-MM-DD COMMIT=XXXX make binary -j8
-        mv node-v15.0.0-nightlyYYYY-MM-DDXXXX-*.tar.gz node-corepack-${{matrix.platform[0]}}-x64.tar.gz
+      - name: "Build Node with corepack master"
+        run: |
+          corepack yarn install --immutable
+          corepack yarn pack
```

Comment:
Why do you call `corepack yarn` rather than just `yarn` (same in the other places)?

Reply:
To make it explicit that we want to use the Corepack version of Yarn, in case Yarn v1.x is installed on the runner. Also, if someone is using this file to know what command to use to run the tests, it's the most likely command to succeed.

Reply:
In that case, perhaps calling `corepack enable` (or perhaps `sudo corepack enable`) beforehand would be better - from a CLI user experience point of view, I'd prefer if Corepack didn't require any change, to guarantee reduced friction.

Reply:
I agree that might be desirable for CLI usage, I'm doubtful it's a valid point in a CI context. The tests already ensures that calling the `corepack` binary is optional once `corepack enable` has been run, right?

Adding a `corepack enable` step in CI seems "wasteful" to me, but if you feel strongly about this, that's fine by me. wdyt?